### DISCUSSION TITLE
Is it possible to configure links to open in the desktop app?

### DISCUSSION THREAD

**Discussion by gnapse** [outline/outline#8126] [General] [answered]
If we're in a browser (or any other app), and there's a link to a documentation page in Outline, is it possible to configure things so that clicking that link will open it in the desktop app, instead of in the browser?

And in particular, how to do this when using a custom domain?

**Comment by tommoor** [comment, marked as answer]
It's not possible, but you can use an app like Choosy (https://choosy.app/) to achieve this by directing links to the custom domain to the app instead. The app protocol is `outline://`, so just replace `https://` in any link
